.travis.yml

```diff
@@ -7,6 +7,23 @@ matrix:
   include:
   - jdk: oraclejdk7
     script: mvn -q deploy --settings .travis-settings.xml -Dno.gem.deploy=true
+  - jdk: oraclejdk7
+    env: ANDROID_EXAMPLES=true
```

Comment:
The intent of the environment variable is to make it obvious on the Travis build page, which of the oraclejdk7 jobs that is running the Android examples (and which is building, testing and deploying the Cucumber-JVM modules). It actually is not used in the execution of the job.